### PR TITLE
Add table plugin example

### PR DESCRIPTION
**Pull request by akmarzhan1**
- [x] created a CodeSandbox example for the table plugin [here](https://codesandbox.io/s/lexical-table-plugin-example-zd6k44)
- [x] added a new page and the example on the website (under “/docs/demos/plugins/tables”)

Table Plugin

Issue https://github.com/facebook/lexical/issues/2886

### CODE REVIEW

File: packages/lexical-website/docs/demos/plugins/tables.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: "Table Plugin"
+sidebar_label: "Table Plugin"
+---
+
+This page focuses on implementing the table plugin and the code you need to incorporate tables into your editor. You can check out the CodeSandbox directly or view, edit and try out the code in real-time inside the embed. 
+
+The below example has all the basic functionality of the tables (e.g., inserting and removing columns and creating headers). Note that the header text is bold, and you can either hardcode it to be simple text or import the relevant styles from Lexical (e.g., italic, bold, underlined) to change the format. There are quite some files to be included and CSS, but they cover most of the table functionality. 
+
+<iframe src="https://codesandbox.io/embed/lexical-table-plugin-example-zd6k44?fontsize=14&hidenavigation=1&module=/src/Editor.js,/src/plugins/TableToolbar.tsx,/src/plugins/TablePlugin.tsx&theme=dark&view=split"
+     style={{width:100+"%", height:700+"px", border:0, "border-radius": 4+"px", overflow:"hidden"}}
```
Comment (thegreatercurve) [inline packages/lexical-website/docs/demos/plugins/tables.md:12]:
```suggestion
     style={{width:"100%", height:"700px", border:0, borderRadius: "4px", overflow:"hidden"}}
```

File: packages/lexical-website/docs/demos/plugins/tables.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+     title="lexical-plain-text-example (forked)"
```
Comment (thegreatercurve) [inline packages/lexical-website/docs/demos/plugins/tables.md:13]: Can we maybe rename this to be something like "lexical-plain-text-table-plugin-example"

**Review by thegreatercurve — APPROVED** [review]

File: packages/lexical-website/docs/demos/plugins/tables.md (hunk 2 above)
Comment (akmarzhan1) [reply packages/lexical-website/docs/demos/plugins/tables.md]: Oh, completely missed these, changed the titles for all plugin examples.

**Comment by akmarzhan1** [comment]
@thegreatercurve implemented your suggestions, let me know if I need to fix anything.

**Pull request merged**